src/backend/marsha/core/tests/test_api_video.py

```diff
@@ -7,6 +7,8 @@
 from django.test import TestCase, override_settings
 
 import pytz
+from rest_framework.authtoken.models import Token
+from rest_framework.test import APIClient
```

Comment:
We are using the client available in `TestCase` class, if we choose using this client we should refactor to use only this client.